transport/awslambda/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package awslambda provides AWS lambda Handler transport layer.
```

Comment:
Is there official guidance on capitalization here? I would expect **AWS Lambda handler** but I don't know.

Reply:
Note sure too, but from googling around its more correct with capitalization. Updated here 50e89617c5c7669dd0d47fcd4e9a771734c58c9b.